docs/Tutorials - start here/semantic_segmentation_tutorial.md

```diff
@@ -0,0 +1,42 @@
+# How to Use Innofw Framework for Semantic Segmentation

```

Comment:
In a nutshell, tutorial is a document that will guide beginner through the project to reach same result. Tutorial should cover steps from A-Z to solve certain problem. In case that information can be duplicated it should use references. So I suggest you to try to train a segmentation model on your own. If you have problems then you should ask me. But in the progress keep track of steps you do.